libs/libc/string/lib_strerror.c

```diff
@@ -392,5 +399,12 @@ FAR char *strerror(int errnum)
 #else
   UNUSED(errnum);
 #endif
-  return "Unknown error";
+#ifdef CONFIG_LIBC_STRERROR_ERRNUM
```

Comment:
let's place this block under `#else` of `ifdef CONFIG_LIBC_STRERROR`

Reply:
Although I let `config LIBC_STRERROR_ERRNUM`'s default be `!LIBC_STRERROR`, but I still want to keep the possibility that they're enabled together.
It's very useful when `LIBC_STRERROR` is not enabled, but it can still help a little when enabled together with `LIBC_STRERROR` if unknown error really happens, so I think keep the chance might be better.

Reply:
Found that `UNUSED(errnum);` may not be under correct condition, so slightly modify it.